src/ground_truth_nsrts.py

```diff
@@ -2107,3 +2110,22 @@ def _get_predicate(base_pred_name: str,
                 f"Unexpected base option name: {base_option_name}")
 
     return nsrts
+
+
+def _get_pddl_env_gt_nsrts(name: str) -> Set[NSRT]:
+    env_base = get_or_create_env(name)
+    env = cast(_PDDLEnv, env_base)
```

Comment:
before casting, should we assert isinstance(env_base, _PDDLEnv)?

Reply:
ignoring because of the weirdness with isinstance